js/core.js

```diff
@@ -3777,6 +3835,18 @@
                 if(jsonAnnotation.to_name){
                     this.to_name = jsonAnnotation.to_name;
                 }
+
+                if(jsonAnnotation.to_comment){
+                    // check for null, false, empty string when digesting comment for first time
+                    this.to_comment = _processModelComment(jsonAnnotation.to_comment);
+                    if (jsonAnnotation.to_comment_display) this.to_comment_display = jsonAnnotation.to_comment_display;
```

Comment:
I think you should set a default value for `to_comment_display` and `from_comment_display`.